spa/src/components/account/Verify/Verify.js

```diff
@@ -0,0 +1,113 @@
+import { useState, useReducer, useMemo } from 'react';
+import { useLocation } from 'react-router-dom';
+import * as S from './Verify.styled';
+import { Link } from 'react-router-dom';
+
+// AJAX
+import axios from 'ajax/axios';
+import { VERIFY_EMAIL_REQUEST_ENDPOINT } from 'ajax/endpoints';
+
+import logo from 'assets/images/logo-nre.png';
+import sendIcon from 'assets/icons/send.png';
+import draftIcon from 'assets/icons/draft.png';
+import readIcon from 'assets/icons/mark_read.png';
+
+// Analytics
+import { useConfigureAnalytics } from 'components/analytics';
+
+import { useUserDataProviderContext } from 'components/Main';
+import PageTitle from 'elements/PageTitle';
+
+// State management
+import fetchReducer, { initialState, FETCH_START, FETCH_SUCCESS, FETCH_FAILURE } from 'state/fetch-reducer';
+
+import {
+  VERIFIED_RESULTS_ACCEPTED_VALUES,
+  VERIFIED_HELP_EMAIL,
+  RESEND_VERIFICATION_SUCCESS_TEXT
+} from 'constants/textConstants';
+
+const Mailto = ({ mailto, label }) => {
+  return (
+    <Link
+      to="#"
+      onClick={(e) => {
+        window.location.href = mailto;
```

Comment:
Would probably add the `mailto` string here
```suggestion
const Mailto = ({ mailto, label }) => {
  return (
    <Link
      to="#"
      onClick={(e) => {
        window.location.href = `mailto:${mailto}`;
```

Reply:
done